src/components/site/SiteHeader.md

```diff
@@ -1,3 +1,12 @@
+Site header is just an empty contanier but we have added some components to display how it will hold certain items 
 ```jsx
-<SiteHeader />
+import LogoType from "./LogoType";
+import SiteStatus from "./SiteStatus";
+
+<SiteHeader>
+    <LogoType LogoHeader="Baltimore County" LogoText="Government">
+    </LogoType>
+    <SiteStatus StatusName="BETA">
```

Comment:
general comments props names are typically lower case. I can't find any standard that says this, but it can't find any instance where they use that casing in their [docs](https://reactjs.org/docs/components-and-props.html).

In this particular case, i think we can just use 'status' to simplify.

Reply:
fixed in logotype as well